emperor/support_files/js/emperor.js

```diff
@@ -739,6 +738,14 @@ function colorChangedForTaxaSpheres(color){
 	}
 }
 
+function colorChangedForTaxaLabels(color){
+        // get the taxonomic assignments and append '_taxalabel' to
+        // retrieve all the labels belonging to a sphere in the plot
+        for(var key in g_taxaPositions) {
```

Comment:
Functionality works fine and I can make this changes. Gonna erase this comments and merge this PR.